test/txstart-test.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('bsert');
+const fs = require('fs');
+const {resolve} = require('path');
+const FullNode = require('../lib/node/fullnode');
+const TX = require('../lib/primitives/tx');
+const Input = require('../lib/primitives/input');
+const Output = require('../lib/primitives/output');
+const MemWallet = require('./util/memwallet');
+const AirdropProof = require('../lib/primitives/airdropproof');
+const Block = require('../lib/primitives/block');
+const Address = require('../lib/primitives/address');
+const Script = require('../lib/script/script');
+const common = require('../lib/blockchain/common');
+const ownership = require('../lib/covenants/ownership');
+const VERIFY_NONE = common.flags.VERIFY_NONE;
+
+const node = new FullNode({
+  memory: true,
+  network: 'regtest',
+  plugins: [require('../lib/wallet/plugin')]
+});
+
+const RESET_TXSTART = node.network.txStart;
+
+// Ease up that mempool
+node.mempool.options.minRelay = 0;
+node.mempool.options.rejectAbsurdFees = false;
+node.miner.options.minWeight = 1000000;
+
+// Use a valid, working airdrop proof
+const FAUCET_PROOF_FILE = resolve(__dirname, 'data', 'faucet-proof.base64');
+const rawFaucetProof =
```

Comment:
Echoing boyma's comment on the goosig PR, this could fit on one line if you call it `raw` and change `FAUCET_PROOF_FILE` to `PROOF_FILE`

Reply:
thanks, force pushed